### ISSUE TITLE
Windows Xp Crash

### ISSUE THREAD

**Issue by dimaspaf14** [yapb/yapb#311]
**Describe the bug**
Installed successful on windows xp sp3, but when i use the menus, then i choose command menu > 1.make double jump, the game crashes

**Expected behavior**
Make double jump like normally

**Video**
Later?..

**Environment details:**
 - OS: Windows Xp Sp3
 - Engine: regame/vanilla? ( i forgor)
 - GameDLL: regame/vanilla? ( i forgor too)
 - Metamod: an metamod (forgor
 - Bot Version: newest
 - Game mode: de_

Additionally, please provide list of AMX Mod X plugins: ``amxx plugins`` and list of Metamod plugins: ``meta plugins``.
Amxx plugins:
(too many, i forgor all of em, ill add later)

Meta plugins:
Some default amxx libraries (like hamsandwich, etc)
The metamod itself
& obviously, Yapb

**Additional context**
Im not using my notebook right now, soo ill add some info later..

I also has feature request, ill describe here cuz i dont want to open new issue, its just simple, can you add a feature that make the bots can reply to player chat like "realbot" feature, example:
Player: "f*ck"
Bots: "watch your mouth, Player!"
Another bots: "someone is being toxic here!"

Thanks @jeefo for making the bots compatible with the old windows xp

**Comment by Vladislav4KZ** [comment]
Can you write a specific version of the bot, not the "newest" one?

**Comment by Vladislav4KZ** [comment]
And you did not add other bots to the game in parallel with yapb?

I literally wrote about this bug yesterday, and it was fixed (did not check yet).

Download the latest binary from GitHub Actions

**Comment by dimaspaf14** [comment]
> Can you write a specific version of the bot, not the "newest" one?

V.4.2.637

> And you did not add other bots to the game in parallel with yapb?

Well, i did, i add zbot bots separately with yapb (:

**Comment by Vladislav4KZ** [comment]
The YaPB command menu from the latest build from GitHub Actions works good for me, with other bots

**Comment by Vladislav4KZ** [comment]
> I also has feature request, ill describe here cuz i dont want to open new issue, its just simple, can you add a feature that make the bots can reply to player chat like "realbot" feature, example:
> Player: "f*ck"
> Bots: "watch your mouth, Player!"
> Another bots: "someone is being toxic here!"


In the yapb chat config, there are already replies for this word, if you want, you can edit the en_chat.cfg file yourself

**Comment by dimaspaf14** [comment]
> The YaPB command menu from the latest build from GitHub Actions works good for me, with other bots

Did you use windows xp?, cuz the problem is on my windows xp

> In the yapb chat config, there are already replies for this word, if you want, you can edit the en_chat.cfg file yourself

Ok, ill try it

> try latest build from GHA

What is GHA?, is that the "config" folder from master branch?

**Comment by Vladislav4KZ** [comment]
> Did you use windows xp?, cuz the problem is on my windows xp

I'm using Windows 10, but the problem is not in the version of Windows, you need to update the bot.

> What is GHA?

I have already said several times, this is GitHub Actions. Find the Actions tab in this repository

> is that the "config" folder from master branch?
No, you are talking nonsense. This folder contains configs

**Comment by dimaspaf14** [comment]
> try latest build from GHA

Already tried, now it works, thanks

**Comment by jeefo** [comment]
Assume everything is fine!
